jsonobject/containers.pyx

```diff
@@ -113,6 +113,9 @@ class JsonArray(list):
         for _ in range(j - i):
             self.pop(i)
 
+    def __iadd__(self, b):
+        self._obj = self._obj + b
```

Comment:
I'm not sure this is right. Here's my logic for you to audit. When python sees
```
x += y
```
it translates that to
```
x = x.__iadd__(y)
```
(if x has an `__iadd__` method). In this case `__iadd__` doesn't return anything so while the original object x referred to gets mutated, the variable x ends up with the value `None`.

I think the simplest way to implement `__iadd__` for a JsonArray would be to emulate the update-in-place behavior of `list.__iadd__` and go with 

```
def __iadd__(self, b):
    self.extend(b)
    return self
```

Reply:
excellent points, both of these.  Just made those suggested changes